src/components/Text.tsx

```diff
@@ -348,3 +349,24 @@ export function SocialLink({ url, children }: ChildrenProp & { url: string }) {
     </a>
   )
 }
+
+export const highlightedText = (bold?: boolean, center?: boolean) =>
+  classnames(
+    width('w-fit'),
+    textColor('text-primary-dark'),
+    fontWeight(bold ? 'font-bold' : 'font-semibold'),
+    fontFamily(bold ? 'font-primary' : undefined),
+    fontSize(bold ? 'text-base' : 'text-sm'),
+    lineHeight(bold ? 'leading-6' : 'leading-5'),
+    textAlign(center ? 'text-center' : 'text-left')
```

Comment:
Can we use `{}` here as much as we can, you don't need the `text-left`, it's default, thats why I want you to use the object here, to avoid typing `undefined` and explicit classnames that are default (like text-left, maybe `leading-5` is default too)

Reply:
Updated where it was possible